tests/blockchain.test.js

```diff
@@ -96,12 +96,32 @@ describe('Blockchain class', function() {
     		blockchain = createBlockchainWithTx();
     		blockchain.chain[1].transactions[0].amount = 897397;
     		assert(!blockchain.isChainValid());
-    	});
+		});
 
     	it('should fail when a block has been changed', function(){
     		blockchain = createBlockchainWithTx();
     		blockchain.chain[1].timestamp = 897397;
     		assert(!blockchain.isChainValid());
     	});
-    });
+	});
+	
+	describe("getAllTransactionsForWallet", function(){
+		it("should get all Transactions for a Wallet", function() {
+			const validTx = createSignedTx();
+    		blockchain.addTransaction(validTx);
+    		blockchain.addTransaction(validTx);
+
+			blockchain.minePendingTransactions("b2");
+			blockchain.addTransaction(validTx);
+			blockchain.addTransaction(validTx);
+			blockchain.minePendingTransactions("b2");
+
+			for (const trans of blockchain.getAllTransactionsForWallet("b2")) {
```

Comment:
I guess we should also check the length of the array that is returned by `blockchain.getAllTransactionsForWallet("b2")`. It should only contain 2 transactions.

Reply:
ok will add